### DISCUSSION TITLE
Expand-all in new website

### DISCUSSION THREAD

**Discussion by nthapaliya** [tteck/Proxmox#3036] [Ideas]
Right now, I can expand a single category, and when I expand another, the first one closes. Occasionally I would like to see all the scripts listed in one go, so it would be nice to have an 'expand-all' button somewhere in the interface.

**Comment by tteck** [comment]
This is a personal preference, and while I understand the desire to have an 'expand-all' button to view all the scripts at once, others might prefer the current setup where expanding a new category closes the previous one. This method can help keep the interface cleaner and more manageable for users who prefer to focus on one category at a time. Currently, there are no plans to implement an 'expand-all' feature.
